charts/stable/plaxt/CHANGELOG.md

```diff
@@ -2,24 +2,6 @@
 
 
 
-## [plaxt-0.0.21](https://github.com/truecharts/charts/compare/plaxt-0.0.20...plaxt-0.0.21) (2022-08-30)
-
-### Chore
-
-- update helm chart common to v10.5.5 ([#3626](https://github.com/truecharts/charts/issues/3626))
-
-
-
-
-## [plaxt-0.0.20](https://github.com/truecharts/charts/compare/plaxt-0.0.19...plaxt-0.0.20) (2022-08-29)
-
-### Chore
-
-- update helm general non-major ([#3619](https://github.com/truecharts/charts/issues/3619))
-
-
-
-
```

Comment:
Id prefer to keep the changelog complete

Reply:
im confused on this one, is the changelog suppose to be empty? i never touch this file.

Reply:
No, it should have all previous changes.
Seems like somehow you deleted the last two logs.
Just add them back.

Reply:
i checked and it was still normal, idk why its empty here but it never changed in the pr when checking it the src.

Reply:
this wouldnt have an issue, right? `This branch is out-of-date with the base branch`